### PR TITLE
Added Example

### PR DESCRIPTION
**Pull request by alexliubj**
*Issue #, :*
Added example project

*Description of changes: *

Added steps on how to start, build and set up Smoke-frame work with a simple http post request.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Example/Tests/SmokeAPITests/SmokeAPITests.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import class Foundation.Bundle
+
+final class SmokeAPITests: XCTestCase {
+    func testExample() throws {
+        // This is an example of a functional test case.
+        // Use XCTAssert and related functions to verify your tests produce the correct
+        // results.
+
+        // Some of the APIs that we use below are available in macOS 10.13 and above.
+        guard #available(macOS 10.13, *) else {
```
Comment (tachyonics) [inline Example/Tests/SmokeAPITests/SmokeAPITests.swift]: Is there a reason the tests are just for macOS?

File: Example/Sources/SmokeAPI/ExampleOutput.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
+//
+//  ExampleOutput.swift
+//  
+//
+//  Created by Alex Liu on 2018-10-24.
+//
```
Comment (tachyonics) [inline Example/Sources/SmokeAPI/ExampleOutput.swift]: Would you be able to remove the `created by` part of the header and add the license header that is on other files.

Comment (alexliubj) [reply Example/Sources/SmokeAPI/ExampleOutput.swift]: Thanks @tachyonics for your review.
Updated my file header.

File: Example/Tests/SmokeAPITests/SmokeAPITests.swift (hunk 1 above)
Comment (alexliubj) [reply Example/Tests/SmokeAPITests/SmokeAPITests.swift]: Deleted my test case which were generated automatically by system.

File: Example/Sources/SmokeAPI/ExampleOutput.swift (hunk 2 above)
Comment (alexliubj) [reply Example/Sources/SmokeAPI/ExampleOutput.swift]: hmm, somehow the latest changes are not showing up in this existing merge request.

Comment (tachyonics) [reply Example/Sources/SmokeAPI/ExampleOutput.swift]: Sorry for not getting back sooner. I am looking at creating a separate examples package with an example that hows a REST-like interface and using some of the other smoke-* packages. Is it alright if I resolve this pull request in favour of that work?

Comment (tachyonics) [reply Example/Sources/SmokeAPI/ExampleOutput.swift]: Resolving this as mentioned it will be superseded by a separate examples package.

**Comment by tachyonics** [comment]
Resolving this as mentioned it will be superseded by a separate examples package.

**Pull request closed without merging**